test/agentchat/test_async_get_human_input.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
+import autogen
+import pytest
+from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST
+
+
+@pytest.mark.asyncio
```

Comment:
Do you know how to get this test running by pytest? I tried to install pytest-asyncio but the test is still skipped.

Reply:
I did 
pytest test/agentchat/test_async_get_human_input.py

Reply:
@sonichi I was able to pip install pytest-asyncio and then was able to run the run. @bonadio I noticed that the coverage went down. @sonichi did you notice that as well?